doc/intro.rst

```diff
@@ -138,15 +141,16 @@ Weather simulators
 
 A code snippet of starting the ``Weather`` simulator generated from the ``Weather.xmi`` file
 with a ``SimControl`` instance using the ``tango_launcher.py`` script.
+Change directory to tango_simlib/examples/
 
 .. code-block:: bash
 
     $ tango-simlib-tango-launcher --name mkat_sim/weather/1 --class Weather\
                           --name mkat_simcontrol/weather/1\
                           --class WeatherSimControl\
-                          --server-command tango-simlib-weather-xmi-DS --port 0\
+                          --server-command Weather1.py --port 0\
```

Comment:
The actual Python files in the examples folder don't have a capital "W".  For example, `weather1.py`.  Same for the next example.